### PR TITLE
PD-6747 CI loop

### PR DESCRIPTION
**Pull request by rvanburen-r7**
Updated dependencies, linting rules, and license.
Added coverage badge to README

### CODE REVIEW

**Comment by fmitchell-r7** [comment]
It looks like [istanbul](https://gotwarlost.github.io/istanbul/) isn't included in the `devDependencies` so the CI loop is failing.

File: package.json [hunk 1]
```diff
@@ -7,9 +7,12 @@
...
   },
   "scripts": {
+    "start": "node ./bin/server.js",
     "test": "mocha",
-    "lint": "eslint bin/** lib/** test/**",
-    "install": "cd pkcs7 && bundle install"
+    "cover": "rm -Rf coverage/* && istanbul cover _mocha -- -R spec",
+    "lint": "eslint bin/* lib/*.js lib/**/*.js test/*.js test/**/*.js",
+    "report": "coveralls < ./coverage/lcov.info",
+    "warden-mock": "node ./test/bin/warden.js"
```
Comment (dgreene-r7) [inline package.json]: Copy pasta?

Comment (rvanburen-r7) [inline package.json]: copy pasta but forgot to save changes made to the script. 
updated script just pushed.

**Comment by dgreene-r7** [comment]
:shipit:

**Pull request merged**